src/content/reference/react/useOptimistic.md

```diff
@@ -143,3 +148,67 @@ export async function deliverMessage(message) {
 ```
 
 </Sandpack>
+
+---
+
+### Optimistically update the UI while waiting for a network request {/*optimistically-update-while-waiting-for-network*/}
```

Comment:
Hmm I may be forgetting something but what was the reasoning for us to add this example? It seems quite similar to the form action example above, the only difference being that it isn't in a form? I think we'd want to add some description here of why this example is worth highlighting. 

In addition, I think it'd be valuable to expand the example and show a failed optimistic update.

Reply:
Yes it is similar but wanted to showcase another example that isn't using a form. I agree on adding in a failure example and have it as a followup

Reply:
Can we add a line about that?
Something like 
```suggestion
### Optimistically update the UI while waiting for a network request {/*optimistically-update-while-waiting-for-
network*/}

`useOptimistic` can also be used outside of forms, for any network request but within a transition.
```